conseil-lorre/src/main/resources/application.conf

```diff
@@ -30,6 +30,14 @@ lorre {
     }
   ]
 
+  # feature flags: select if some functionality should be turned off
+  enabled-features {
+    block-rights-fetching-is-on: true
+    block-rights-fetching-is-on: ${?CONSEIL_LORRE_BLOCK_RIGHTS_FETCHING_ENABLED}
+    fork-handling-is-on: true
```

Comment:
Please set the default to false.

Reply:
On second thought, we cannot set this to false unless we also relax the db constraints.

Reply:
I'd say that the flag doesn't change much in terms of consistency guarantees for the DB... yet it removes additional sources of errors when the fork happens, until I solve the bug